[chore] removes trailing forward slash from admin route

koyagabriel opened the pull request:
## What does this PR do?

Adjusts the route to admin page

## Description of Task to be completed?

Remove the trailing forward slash from the admin route in the project's route map

## Relevant Pivotal Story

N/A

Commit:
[chore] removes trailing forward slash from adim route

BolajiOlajide commented on server/lostandfound/urls.py:
```diff
@@ -18,5 +18,5 @@
 admin.site.site_header = "Lost and Found Administration"
 urlpatterns = [
-    url(r'^admin/', admin.site.urls),
+    url(r'^admin', admin.site.urls),
```
I'm not sure if this addresses the problem of trailing slash. Can you clarify what happens when you try to access `localhost:8000/admin/`. If it actually locates the admin page.

koyagabriel replied:
for `localhost:8000/admin/`, the page wont be found based on the new changes made on the route. User tend to input this url `localhost:8000/admin` more than its counterpart, but we can still take care of the edge case you highlighted by including a second route to take care of the odds.

Commit:
[bug] fix the admin route
added a new admin route to cater for a possible edge case
in the admin's url route mapping

BolajiOlajide commented on server/lostandfound/urls.py:
```diff
@@ -19,4 +19,5 @@
 admin.site.site_header = "Lost and Found Administration"
 urlpatterns = [
     url(r'^admin', admin.site.urls),
+    url(r'^admin/', admin.site.urls),
```
So instead of having two routes to the admin page, you can just add a `APPEND_SLASH` setting to the settings.py as shown in the attached picture.
That takes care of any routes that the user may want to add a trailing slash.

<img width="533" alt="screen shot 2017-04-02 at 2 01 01 pm" src="https://cloud.githubusercontent.com/assets/25608335/24587350/e8891d46-17ac-11e7-8ce5-943c4310699d.png">

BolajiOlajide replied:
All it takes is to set it to `True` i think by default it was `False`. That fixes this issue. So no need to have two routes.

It wont make sense to have two routes for each route because of trailing slash.

koyagabriel replied:
Yep, you right. Thanks for the insight.

koyagabriel replied:
It seems it only works for the default route to the index page `localhost:8000`, because with the value of `APPEND_SLASH` set to `True`, `localhost:8000/admin/` will still return a `404 status code`

koyagabriel replied:
So i guess i will still have to inject back the second route for the other admin url, or what do you feel.

koyagabriel replied:
neva mind, i have resolved it. Your insight was still right.

BolajiOlajide replied:
Awesome. So i guess this is cleared for merging.

koyagabriel replied:
yep

Commit:
[chore] removes redundant route

Commit:
[bug] fixes trailing slash admin route
